data/human/news.txt

```diff
@@ -1183,7 +1183,7 @@ news "human farming in pirate space"
 			"I came out here because I liked the company of animals more than people like you. Go away."
 			"Shocked to see some normal farmers wandering 'round here? Just because we live on a lawless world doesn't mean it's kill or be killed."
 			"My son was enslaved by a gang of pirates. Not much one can do around here to prevent that."
-			"My daugher was held hostage by a gang of pirates until I coughed up some money for them. It's inevitable in this social climate."
+			"My daughter was held hostage by a gang of pirates until I coughed up some money for them. It's become such a common occurance that I have to keep a stockpile of credits just in case."
```

Comment:
```suggestion
			"My daughter was held hostage by a gang of pirates until I coughed up some money for them. It's become such a common occurrence that I have to keep a stockpile of credits just in case."
```